app/models/program.rb

```diff
@@ -113,4 +118,23 @@ def create_difficulty_levels
                                                 color: '#EF6E69')
     true
   end
+
+  ##
+  # Check if languages string has the right format. Used as validation.
+  #
+  def check_languages_format
+    # All white spaces are removed to allow languages to be separated by ',' and ', ' but only the first option is saved
```

Comment:
This comment should be one line lower, as it refers to L128, not L127
What do you mean only the first option is saved?